pkg/scalers/aws_sqs_queue_scaler_test.go

```diff
@@ -240,6 +246,19 @@ var testAWSSQSMetadata = []parseAWSSQSMetadataTestData{
 		testAWSSQSAuthentication,
 		false,
 		"properly formed queue and region"},
+	{map[string]string{
+		"queueURLFromEnv": "QUEUE_URL",
+		"queueLength":     "1",
+		"awsRegion":       "eu-west-1"},
+		testSQSResolvedEnv,
+		false,
+		"properly formed queue loaded from env"},
+	{map[string]string{
+		"queueLength": "1",
+		"awsRegion":   "eu-west-1"},
+		testSQSResolvedEnv,
+		true,
+		"missing queue url from both queueURL and queueURLFromEnv"},
```

Comment:
```suggestion
	{map[string]string{
		"queueURLFromEnv": "QUEUE_URL",
		"queueLength":     "1",
		"awsRegion":       "eu-west-1"},
		testAWSSQSAuthentication,
		false,
		"properly formed queue loaded from env"},
		
		
		
	{map[string]string{
		"queueLength": "1",
		"awsRegion":   "eu-west-1"},
		testAWSSQSAuthentication ,
		true,
		"missing queue url from both queueURL and queueURLFromEnv"},
```

https://github.com/kedacore/keda/blob/708890964f00c169ba990fa5fd497b47893a4ded/pkg/scalers/aws_sqs_queue_scaler_test.go#L258

Here (on line258) use `testSQSResolvedEnv` instead of `testAWSSQSAuthentication` , and similarly for the other test as well.

Reply:
I've made an update to introduce `ResolvedEnv` in the testData struct to allow testing for the case where it's the env variable is an empty string.